app/assets/stylesheets/styles/components/guides/ui/_actions_modal.css.scss

```diff
@@ -0,0 +1,30 @@
+.actions-modal {
+  -webkit-transition: opacity 200ms ease-in;
```

Comment:
Properties should be ordered pointer-events, -moz-transition, -webkit-transition, transition<br>Avoid vendor prefixes.